### PR TITLE
Refactored mesos documentation

### PR DESCRIPTION
**Pull request by djosborne**

### CODE REVIEW

File: docs/mesos/README.md [hunk 1]
```diff
@@ -7,16 +7,24 @@
...
+Questions? Contact us on the #Mesos channel of [Calico's Slack](https://calicousers-slackin.herokuapp.com/).
 
-Calico support is under development. Use the following information to ensure you choose the right version:
-- Calico fully supports and recommends Mesos 0.26
-- Calico supports Mesos 0.25, but we recommend against using it as there aren't any Frameworks (including Marathon) which support the Networkinfo specs from 0.25 (which were modified for 0.26)
-- Calico works on Mesos 0.24, but only as a proof of concept, and is no longer supported.
+### Mesos Version Compatability
+Calico support is actively being developed. Use the following information to ensure you choose the right version:
+- **Mesos 0.26:** Recomended. Full Calico Support for the future.
```
Comment (robbrockbank) [inline docs/mesos/README.md]: m -> mm

File: docs/mesos/ManualInstallCalicoMesos.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+## 7. Launch Mesos-Slave 
+```
+$ sudo ETCD_AUTHORITY=<ETCD-IP:PORT> /usr/local/sbin/mesos-slave \
+--master=<MASTER-IP:PORT> \
+--modules=file:///calico/modules.json \
+--isolation=com_mesosphere_mesos_NetworkIsolator \
+--hooks=com_mesosphere_mesos_NetworkHook
+```
+We provide the `ETCD_AUTHORITY` environment to variable here to allow the  `calico_mesos` plugin to function properly when called by `mesos-slave`. Be sure to replace it with the address of your running etcd server.
```
Comment (robbrockbank) [inline docs/mesos/ManualInstallCalicoMesos.md]: to variable -> variable

File: docs/mesos/ManualInstallCalicoMesos.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.13.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Manually Install Calico and Mesos
+This tutorial will walk you through installing Mesos, Netmodules, and Calico onto a Centos host to create a Mesos Slave compatible with Calico.
```
Comment (TrimBiggs) [inline docs/mesos/ManualInstallCalicoMesos.md]: You also need:
- an ETCD instance running and accessible
- Docker installed on your Centos host

**Comment by TrimBiggs** [comment]
Closing in favor of #723, which includes these changes and additional updates.

**Pull request closed without merging**